src/Utopia/Messaging/Adapters/Email/Sendgrid.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Utopia\Messaging\Adapters\Email;
+
+use Utopia\Messaging\Messages\Email;
+use Utopia\Messaging\Adapters\Email\Email as EmailAdapter;
+
+class Sendgrid extends EmailAdapter
+{
+    public function __construct(
```

Comment:
Is this syntax new?!

Reply:
Yep it was added in PHP 8, this is [constructor property promotion](https://www.php.net/manual/en/language.oop5.decon.php#language.oop5.decon.constructor.promotion). Just short-hand for a property that is assigned via constructor

Reply:
Very cool.